templates/common/infra/bicep/core/host/container-apps-auth.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Container Apps Auth Config using Microsoft Entra as Identity Provider.'
+
+@description('The name of the container apps resource within the current resource group scope')
+param name string
+
+@description('The client ID of the Microsoft Entra application.')
+param clientId string
+
+@description('The name of the Container Apps secret that contains the client secret of the Microsoft Entra application.')
+param clientSecretName string
+
+@description('The OpenID issuer of the Microsoft Entra application.')
```

Comment:
Since this module isn't really a host itself I think it would fit better in the `security` folder with our other security based modules.

Reply:
I put it into host as it's directly related to Container Apps (and from the ARM perspective, is a part of the Container App RP, not a separate service). It felt more similar to templates like appservice-appsettings.bicep which configures a given aspect of App Service/Container Apps.